rskj-core/src/main/java/co/rsk/config/BridgeRegTestConstants.java

```diff
@@ -159,6 +159,8 @@ public BridgeRegTestConstants(List<BtcECKey> federationPublicKeys) {
         oldFederationAddress = "2N7ZgQyhFKm17RbaLqygYbS7KLrQfapyZzu";
 
         minimumPegoutValuePercentageToReceiveAfterFee = 20;
+
+        numberOfBlocksBetweenPegouts = 18; // 3 hours of BTC blocks (considering 1 block every 10 minutes)
```

Comment:
In the case of regtest maybe a small value would be a good idea, so we can quickly test pegouts. I'd say around 50 blocks

Reply:
In regtest the mining is really fast, we can also disable automatic mining and advance the network at demand. This is useful in integration tests for example